Update a python-binding test script

nicelhc13 opened the pull request:
Replace algorithms of SSSP and BFS with asynchronous on road network.
This commit also changes pagerank parameters.

ddn0 commented:
There was some previous discussion about `#pragma once` versus `#ifndef` I don't really want to open that can of worms yet. The goal of this PR is just to ensure consistency when a file does use `#ifndef`

tylershunt commented:
> Another option would be to check if the ifdef macro matches the define
> macro as a sign of guard. But a not uncommon error in header guards is
> for those two macros to be different. Given the infrequency of valid
> uses of this pattern, I opted to treat unequal macros as an error.

I wholeheartedly support this.

ddn0 commented:
> I'm assuming by your comment that you did not change the files with "pragma once". I didn't notice those files in this commit.

Yup. I left the `#pragma once` files alone.

> The only comment is that the last line of the file might have previous (non-sanitized) preprocessor defined names. For example, in DistributedGraph.h:

Let me quickly strip those out.

arthurp commented on scripts/bench_python_cpp_algos.py:
```diff
@@ -83,6 +91,7 @@ def run_sssp(property_graph: PropertyGraph, input_args, source_node_file):
 
         for source in sources:
             with time_block(f"sssp on {source}"):
+                source = source.strip("\n")
```
I'm not sure what's going on here. source shouldn't be a string here.

nicelhc13 replied:
Ah, yes. I have separate version of this script and it was string. My bad! Thank you for pointing this out!

arthurp reviewed (changes requested):
One weird issue that could cause problems, so I want to fully understand it.

arthurp commented on scripts/bench_python_cpp_algos.py:
```diff
@@ -128,7 +137,7 @@ def run_jaccard(property_graph: PropertyGraph, input_args):
 def run_pagerank(property_graph: PropertyGraph, _input_args):
     property_name = "NewProp"
 
-    tolerance = 0.000001
```
This will conflict with #288.

Commit:
Update a python-binding test script

Replace algorithms of SSSP and BFS with asynchronous on road network.
This commit also changes pagerank parameters.

roshandathathri reviewed (approved):
LGTM
